git/git.go

```diff
@@ -115,6 +118,20 @@ func pull(url, repoPath string) ([]string, error) {
 	return updatedFiles, nil
 }
 
+func fetchAll(repoPath string) error {
+	commandArgs := generateGitArgs(repoPath)
+	configCmd := []string{"config", "remote.origin.fetch", "+refs/heads/*:refs/remotes/origin/*"}
+	if _, err := utils.Exec("git", append(commandArgs, configCmd...)); err != nil {
+		return xerrors.Errorf("error in git config: %w", err)
```

Comment:
Do we still need xerrors in Go 1.13?
```suggestion
		return fmt.Errorf("error in git config: %w", err)
```

Reply:
`errors` supported `Wrap`, but unfortunately it doesn't support stack trace yet.
https://play.golang.org/p/xwGyjc6FyRL

This stack trace is useful for debugging, especially when someone raises an issue with the error message. So, I prefer using `xerrors` yet.

Reply:
Ah I see. Yup it makes sens.